applications/openshift/master/file_groupowner_etcd_data_dir/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Verify Grop Who Owns The Etcd Database Directory'
+
+description: |-
+    {{{ describe_file_group_owner(file="/var/lib/etcd", group="root") }}}
+
+rationale: |-
+    etcd is a highly-available key-value store used by Kubernetes deployments for
+    persistent storage of all of its REST API objects. This data directory should
+    be protected from any unauthorized reads or writes. It should be owned by
+    <pre>root:root</pre>.
+
+severity: medium
+
+#identifiers:
+#    cce@ocp4:
+
+references:
+    cis: 1.1.12
+
+ocil_clause: '{{{ ocil_clause_file_group_owner(file="/var/lib/etcd", group="root") }}}'
+
+ocil: |-
+    {{{ ocil_file_group_owner(file="/var/lib/etcd", group="owner") }}}
+
+template:
+    name: file_groupowner
+    vars:
+        filepath: /var/lib/etcd
```

Comment:
This is really strange, the rule does not seem to be matching..do you know if the file_groupowner template also works for directories? This is the output I'm getting:
```
Rule^M  xccdf_org.ssgproject.content_rule_file_groupowner_etcd_data_dir
I: oscap: Evaluating XCCDF rule 'xccdf_org.ssgproject.content_rule_file_groupowner_etcd_data_dir'.
I: oscap: Evaluating definition 'oval:ssg-installed_app_is_ocp4:def:1': Red Hat OpenShift Container Platform.
I: oscap: Definition 'oval:ssg-installed_app_is_ocp4:def:1' evaluated as false.
I: oscap: Evaluating definition 'oval:ssg-installed_app_is_ocp4_node:def:1': Red Hat OpenShift Container Platform Node.
I: oscap: Definition 'oval:ssg-installed_app_is_ocp4_node:def:1' evaluated as true.
I: oscap: This rule requires an OCIL check. OCIL checks are not supported by OpenSCAP.
I: oscap: Evaluating definition 'oval:ssg-file_groupowner_etcd_data_dir:def:1': Verify Group Who Owns The Etcd Database Directory.
I: oscap:   Evaluating file test 'oval:ssg-test_file_groupowner_etcd_data_dir:tst:1': Testing group ownership of /var/lib/etcd.
I: oscap:     Querying file object 'oval:ssg-object_file_groupowner_etcd_data_dir:obj:1', flags: 0.
I: oscap:     Creating new syschar for file_object 'oval:ssg-object_file_groupowner_etcd_data_dir:obj:1'.
I: oscap:     Switching probe to PROBE_OFFLINE_OWN mode.
I: oscap:     I will run file_probe_main:
I: oscap:     Opening file '/host/var/lib/etcd'.
I: oscap:     Test 'oval:ssg-test_file_groupowner_etcd_data_dir:tst:1' requires that zero or more objects defined by 'oval:ssg-object_file_groupowner_etcd_da>
I: oscap:     0 objects defined by 'oval:ssg-object_file_groupowner_etcd_data_dir:obj:1' exist on the system.
I: oscap:     No item matching object 'oval:ssg-object_file_groupowner_etcd_data_dir:obj:1' was found on the system. (flag=does not exist)
I: oscap:   Test 'oval:ssg-test_file_groupowner_etcd_data_dir:tst:1' evaluated as true.
I: oscap: Definition 'oval:ssg-file_groupowner_etcd_data_dir:def:1' evaluated as true.
Result^M        pass
```

Reply:
I needed to add a `/` in the end... I re-read the audit command and adjusted for a more specific directory. It should be a better check now.